autogen/docs/headings.html.mustache

```diff
@@ -1,143 +1,148 @@
-        <li><a href="requirements.html" target="entry">System Requirements</a></li>
-        <li><a href="contact.html"      target="entry">Contacting Us</a></li>
-        <li><a href="copyright.html"    target="entry">Copyright / License</a></li>
-      </ul>
-    </div>
-    <hr>
-    <div class="heading-section">
-      <span class="heading-title">Introduction</span>
-      <ul class="heading-items">
-        <li><a href="whatis.html" target="entry">What is NetLogo?</a></li>
-        <li><a href="sample.html" target="entry">Sample Model: Party</a></li>
-      </ul>
-    </div>
-    <div class="heading-section">
-      <span class="heading-title">Learning NetLogo</span>
-      <ul class="heading-items">
-        <li><a href="tutorial1.html" target="entry">Tutorial #1: Models</a></li>
-        <li><a href="tutorial2.html" target="entry">Tutorial #2: Commands</a></li>
-        <li><a href="tutorial3.html" target="entry">Tutorial #3: Procedures</a></li>
-      </ul>
-    </div>
-    <div class="heading-section">
-      <span class="heading-title">Reference</span>
-      <ul class="heading-items">
-        <li><a href="interface.html" target="entry">Interface Guide</a></li>
+  .heading-title {
+    display: block;
+    font-weight: bold;
+    white-space: nowrap;
+  }
+</style>
+<div class="headerbanner">
+  <span class="block offwhitetext xlargefont bold">NetLogo</span>
+  <span class="block offwhitetext xlargefont bold nowrap">User Manual</span>
+  <span class="block offwhitetext largefont bold nowrap">version {{version}}</span>
+  <span class="block offwhitetext bold nowrap">{{date}}</span>
+</div>
+<div style="margin-bottom: 0em; margin-top: .25em;">
+  <span class="heading-title xsmallfont">Release Notes</span>
+  <ul class="small-items">
+    <li><a href="versions.html"     target="entry">What's New?</a></li>
+    <li><a href="requirements.html" target="entry">System Requirements</a></li>
+    <li><a href="contact.html"      target="entry">Contacting Us</a></li>
+    <li><a href="copyright.html"    target="entry">Copyright / License</a></li>
+  </ul>
+</div>
+<hr>
+<div class="heading-section">
+  <span class="heading-title">Introduction</span>
+  <ul class="heading-items">
+    <li><a href="whatis.html" target="entry">What is NetLogo?</a></li>
+    <li><a href="sample.html" target="entry">Sample Model: Party</a></li>
+  </ul>
+</div>
+<div class="heading-section">
+  <span class="heading-title">Learning NetLogo</span>
+  <ul class="heading-items">
+    <li><a href="tutorial1.html" target="entry">Tutorial #1: Models</a></li>
+    <li><a href="tutorial2.html" target="entry">Tutorial #2: Commands</a></li>
+    <li><a href="tutorial3.html" target="entry">Tutorial #3: Procedures</a></li>
+  </ul>
+</div>
+<div class="heading-section">
+  <span class="heading-title">Reference</span>
+  <ul class="heading-items">
+    <li><a href="interface.html" target="entry">Interface Guide</a>
+      <ul>
```

Comment:
The styling of this `<ul>` sticks out a bit in the menu bar. Let's add a class to it with the following style:

```css
  ul.interface-items {
    padding-left: 15px;
    list-style: none;
  }
```
```